src/blas/backends/cublas/cublas_scope_handle.hpp

```diff
@@ -68,12 +68,13 @@ class CublasScopedContextHandler {
     CUcontext original_;
     cl::sycl::context placedContext_;
     bool needToRecover_;
+    cl::sycl::interop_handler& ih;
```

Comment:
how will this work, if there's no `interop_handler` in hipsycl?

Reply:
The idea was to ifdef the headers in cublas_task.hpp, such that this header is not even included in case we are compiling with hipSYCL. 

see: https://github.com/sbalint98/oneMKL/pull/2